src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Step 5: Store your recovery phrase
+
+Some apps will request you to save a secret recovery phrase. Keeping this secret recovery phrase safe is extremely important! Any person who knows this secret recovery phrase can take control of all your accounts generated by these words. Never share this with anyone. This phrase should contain 12 to 24 randomly generated words (the order of the words matters).
+
+Once you have saved your recovery phrase you should see your wallet dashboard with your balance. Check out our guide: [how to use a wallet.](/guides/how-to-use-a-wallet)
+
+<InfoBanner shouldSpaceBetween emoji=":eyes:">
+  <div>Want to learn more?</div>
+  <ButtonLink to="/guides/">
+    See our other guides
+  </ButtonLink>
+</InfoBanner>
+
+### Frequently asked questions
+
+### Are my wallet and my Ethereum account the same?
+No, wallets have the ability to create a new Ethereum account, but the account lives on Ethereum and wallets connect to it. So if you lose your wallet you can still recover your account with the recovery phrase.
+
+### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
+
+No you cannot, you need two different accounts and receiving addresses with a different format altogether. There are many EVM (Ethereum Virtual Machine) compatible tokens & blockchains that you can use with your Ethereum address.
+
+### If I own an ETH address, do I own the same address on other blockchains?
+
+You can use the same address on all EVM compatible blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.
+
+### Is having my own wallet safer than keeping my funds on an exchange?
+
+Yes, this is a much safer option because nobody else will have access to your funds. There are unfortunately many examples of failed exchanges that filed for bankrupcy and lost users' savings. Hacks, frozen accounts or blocked withdrawals are some other common risks. Owning a wallet (with a recovery phrase) is the best way to safeguard your assets. If you have a poorly backed up recovery phrase, you're potentially at higher risk than if you had left your money on the exchange though. Make sure to store your recovery phrase well.
+
+### If I lose my phone/hardware wallet, do I need to use the same wallet app again to recover the lost funds?
+
+No, you can use almost any wallet as the recovery process is standardized. You will have to enter the recovery phrase into any new wallet app, it can be different app from the one you used before. It is almost impossible to recover lost funds without recovery phrase.
```

Comment:
Edited typo and proposed language changes to aid readability:

- No, you can use almost any wallet as the recovery process is _largely_  standardized _whereby only knowledge of the recovery phrase is required_. You will have to enter the recovery phrase into any new wallet app, it can be _a_ different app from the one you used before. It is almost impossible to recover lost funds without _the_ recovery phrase.